.circleci/config.yml

```diff
@@ -528,6 +640,7 @@ jobs:
     resource_class: medium
     <<: *environment
     steps:
+      - *attach_workspace
```

Comment:
Create Instances setups its own environment, no need for it to be dependent on `Seup instance` step and to attach workspace.